heron/cli/tests/python/argparser_unittest.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2016 Twitter. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+''' opts_unittest.py '''
+import unittest2 as unittest
+import heron.cli.src.python.activate as activate
+import heron.common.src.python.argparser as argparser
+import heron.common.src.python.utils.config as config
+from heron.common.src.python.color import Log
+import logging
+import heron.cli.src.python.opts as opts
+import os
+import sys
+#pylint: disable=missing-docstring, no-self-use
+help_epilog = '''Getting more help:
+  heron help <command> Prints help and options for <command>
+
+For detailed documentation, go to http://heronstreaming.io'''
+
+class HeronRCTest(unittest.TestCase):
+  logging.basicConfig(level=logging.WARNING)
+
+  def setUp(self):
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+    self.testrcfile = dir_path + "/.heronrc"
+    pass
+
+
+  def test_parser_commandline(self):
+    sys.argv=[]
+
+    parser = argparser.HeronRCArgumentParser(
+      prog = 'heron',
+      epilog = help_epilog,
+      formatter_class=config.SubcommandHelpFormatter,
+      fromfile_prefix_chars='@',
+      add_help = False,
+      rcfile = self.testrcfile,
+      rccommand = "activate",
```

Comment:
pylint is complaining that it needs one space around assignment. Please advise

pylint --rcfile=tools/python/checkstyle.ini heron/cli/tests/python/argparser_unittest.py